### ISSUE TITLE
Documentation fix

### ISSUE THREAD

**Issue by RyanTG** [muffinista/chatterbot#66]
Assuming this isn't a priority, but I'm still using this gem and thought I'd share that `token` and `secret` are incorrect in the [docs](http://muffinista.github.io/chatterbot/setup.html), and instead it should be `access_token` and `access_token_secret`

````
In a separate YAML config file. Create a file named botname.yaml – the botname part must match your bot’s username EXACTLY. Put the following contents, pasting the credential values that you just generated:

  ---
  :consumer_secret: Consumer Secret (API Secret) GOES HERE
  :consumer_key: Consumer Key (API Key) GOES HERE
  :access_token: Access Token GOES HERE
  :access_token_secret: Access Token Secret GOES HERE
```

**Comment by muffinista** [comment]
Thanks for that. If you want to submit a PR for that I'll be happy to merge
it in, otherwise I'll get to this as soon as I can

**Comment by RyanTG** [comment]
I’ll do a PR. I didn’t realize the docs were part of this repo.

**Comment by muffinista** [comment]
Resolved by #67, thanks!
